src/ConstructiveSolidGeometry/LinePrimitives/Line.jl

```diff
@@ -1,61 +1,10 @@
-struct Line{T,TP,TL} <: AbstractLinePrimitive{T}
```

Comment:
Was there something wrong with the original `Line`? From what I see `Edge` is a replacement of `Line{...,:seg}` and `Line` is a replacement of `Line{...,:inf}`. And now there is no ray. We keep overwriting work.

Reply:
This implementation of the Line goes actually back to the original implementation (current master branch): 
https://github.com/JuliaPhysics/SolidStateDetectors.jl/blob/master/src/Geometries/LinePrimitives/LinePrimitives.jl

Reply:
Ans Line and Edge have different fields so they can't be combined in a parametric type. 
So far we don'n need a ray.